README.md

```diff
@@ -15,7 +15,40 @@ Place your plugin features list.
 
 ## Usage
 
-Include a usage description for your plugin.
+Examples:
+
+```
+"EOX_TAGGING_DEFINITIONS":[
+  {
+    "tag_type":"example_tag_1",
+    "belongs_to": "User",    # default = Site
+    "validate_target_type": "User",
+    "validate_access": "Private",
+    "validate_tag_value": {"in": ['value1, value2']}
+  },
+  {
+    "tag_type":"example_tag_2",
+    "belongs_to": "Site",
+    "validate_target_type": {"object":"CourseOverview"},
```

Comment:
Is this possible {"object": ["CourseOverview", "CourseEnrollments"]} right? @felipemontoya